### ISSUE TITLE
How to change the font in Things theme

### ISSUE THREAD

**Issue by Tu-Zhenzhao** [oleeskild/obsidian-digital-garden#564]
Sorry about asking this stupid question but I cannot find any way to do it online. 

I can easily change font to *PT Serif* in Things theme in local but no matter how I edit the style setting plugin and things's css the font in the webpage always reminds no effect. I am wonder how to edit to change the font in webpage. 

This is my `appearance.json`
```
{
  "theme": "system",
  "cssTheme": "Things",
  "baseFontSize": 18,
  "accentColor": "",
  "interfaceFontFamily": "",
  "textFontFamily": "PT Serif"
}
```

and my style-setting
<img width="785" alt="Screenshot 2024-03-08 at 12 01 59 PM" src="https://github.com/oleeskild/obsidian-digital-garden/assets/58062411/1c927379-65d5-4d9c-8101-d4250c510df2">

Any advice will be appreciated!

**Comment by kerim** [comment]
I couldn't find this information anywhere, but looking around I found this file on github:

[your repository]/src/site/styles/custom-style.scss

I put my custom CSS snippets there and it worked.

**Comment by Tu-Zhenzhao** [comment]
> I couldn't find this information anywhere, but looking around I found this file on github:
> 
> [your repository]/src/site/styles/custom-style.scss
> 
> I put my custom CSS snippets there and it worked.

I found it and make some simple change it works but when I put the `font-family: 'PT Serif', serif;` font setting in here, looks like does not work

**Comment by bakebakebakebake** [comment]
> > I couldn't find this information anywhere, but looking around I found this file on github:
> > [your repository]/src/site/styles/custom-style.scss
> > I put my custom CSS snippets there and it worked.
> 
> I found it and make some simple change it works but when I put the `font-family: 'PT Serif', serif;` font setting in here, looks like does not work

Have you found a way to change the code font now? I've been looking for a solution lately but haven't found one.

**Comment by Tu-Zhenzhao** [comment]
> > > I couldn't find this information anywhere, but looking around I found this file on github:
> > > [your repository]/src/site/styles/custom-style.scss
> > > I put my custom CSS snippets there and it worked.
> > 
> > 
> > I found it and make some simple change it works but when I put the `font-family: 'PT Serif', serif;` font setting in here, looks like does not work
> 
> Have you found a way to change the code font now? I've been looking for a solution lately but haven't found one.

Yes I do. I found where we can change the font. Since I am not changing the code font, so cannot give you directly way to edit it. But here is I change the font hope it is helpful:

1. locate to `src/site/styles/custom-style.scss`

2. Add this code to change the font:
```
@import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');

body, .content, .workspace, .markdown-preview-view, .markdown-source-view {
    font-family: 'PT Serif', serif !important;
}
```

Hope this is helpful.

**Comment by bakebakebakebake** [comment]
> > > > I couldn't find this information anywhere, but looking around I found this file on github:我在任何地方都找不到这些信息，但环顾四周，我在 github 上找到了这个文件：
> > > > [your repository]/src/site/styles/custom-style.scss[您的存储库]/src/site/styles/custom-style.scss
> > > > I put my custom CSS snippets there and it worked.我把我的自定义 CSS 片段放在那里，它工作了。
> > > 
> > > 
> > > I found it and make some simple change it works but when I put the `font-family: 'PT Serif', serif;` font setting in here, looks like does not work我找到了它并进行了一些简单的更改，它有效，但是当我在此处输入 `font-family: 'PT Serif', serif;` 字体设置时，看起来不起作用
> > 
> > 
> > Have you found a way to change the code font now? I've been looking for a solution lately but haven't found one.您现在找到更改代码字体的方法了吗？我最近一直在寻找解决方案，但还没有找到。
> 
> Yes I do. I found where we can change the font. Since I am not changing the code font, so cannot give you directly way to edit it. But here is I change the font hope it is helpful:是的，我愿意。我找到了我们可以更改字体的地方。由于我没有更改代码字体，因此无法直接为您提供编辑它的方法。但这是我更改字体，希望它有帮助：
> 
> 1. locate to `src/site/styles/custom-style.scss` locate 到 `src/site/styles/custom-style.scss`
> 2. Add this code to change the font:添加以下代码以更改字体：
> 
> ```
> @import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');
> 
> body, .content, .workspace, .markdown-preview-view, .markdown-source-view {
>     font-family: 'PT Serif', serif !important;
> }
> ```
> 
> Hope this is helpful.希望这有帮助。

I followed your instructions, but I have a question: in the light theme, the code is duplicated, as shown in the image. 

![image](https://github.com/user-attachments/assets/32df4075-b7cf-49c8-a4bc-44182e8d12da)

Is there a better solution to this issue? 

I would greatly appreciate it if this problem could be resolved.

**Comment by Tu-Zhenzhao** [comment]
My solution is only working for changing body font. But I believe there is place for configuring the background font. You want to find it first in themes css, then overwrite it in customized-style.

**Comment by bakebakebakebake** [comment]
> My solution is only working for changing body font. But I believe there is place for configuring the background font. You want to find it first in themes css, then overwrite it in customized-style.

Okay, thank you anyway.
